pkg/scheduler/framework/interface.go

```diff
@@ -506,10 +506,10 @@ type Framework interface {
 	ListPlugins() map[string][]config.Plugin
 }
 
-// FrameworkHandle provides data and some tools that plugins can use. It is
+// Handle provides data and some tools that plugins can use. It is
 // passed to the plugin factories at the time of plugin initialization. Plugins
 // must store and use this handle to call framework functions.
-type FrameworkHandle interface {
+type Handle interface {
```

Comment:
@alculquicondor @Huang-Wei fyi, this is being renamed.

Reply:
Is there any reason that we must rename the interface? If it is renamed, plugins in [scheduler-plugins](https://github.com/kubernetes-sigs/scheduler-plugins ) and other plugins would also must change when upgrading.

Reply:
golint will fail without this change.
```
Errors from golint:
pkg/scheduler/framework/interface.go:512:6: type name will be used as framework.FrameworkHandle by other packages, and that stutters; consider calling this Handle
```

Reply:
+1 to renaming